### ISSUE TITLE
bug: buttons with missing accessible text

### ISSUE THREAD

**Issue by RitaDee** [open-sauced/app#2028]
### Description

The following three buttons on the notification.tsx file have missing accessible text, making it difficult for screen reader users to understand their purpose:

1. "All" button.
2. "Follows" button.
3. "Reactions" buttons.

### Affected Code
"All" button
```
             <Button
              variant="link"
              className={clsx("hover:text-orange-600", filter === "all" ? "text-orange-600" : "text-light-slate-11")}
              onClick={() => {
                setFilter("all");
                router.push(`/user/notifications?filter=all`);
              }}
            >
              All
           </Button>
```

"Follows" button
```
             <Button
              variant="link"
              className={clsx("hover:text-orange-600", filter === "follow" ? "text-orange-600" : "text-light-slate-11")}
              onClick={() => {
                setFilter("follow");
                router.push(`/user/notifications?filter=follow`);
              }}
            >
              Follows
            </Button>
```

 "Reactions" buttons
```
             <Button
              variant="link"
              className={clsx(
                "hover:text-orange-600",
                filter === "highlight_reaction" ? "text-orange-600" : "text-light-slate-11"
              )}
              onClick={() => {
                setFilter("highlight_reaction");
                router.push(`/user/notifications?filter=highlight_reaction`);
              }}
            >
              Reactions
            </Button>
```

### Proposed Solution
To make these buttons accessible, we need to add accessible text that describes the purpose of each button. A common and effective method is to use the aria-label attribute to provide a descriptive label for each button. Here are the proposed solutions for each button:

"All" Button: Add an aria-label attribute with a value like "Show All Notifications" to describe the button's purpose.

"Follows" Button: Add an aria-label attribute with a value like "Filter Notifications by Follows" to describe the button's purpose.

"Reactions" Button: Add an aria-label attribute with a value like "Filter Notifications by Reactions" to describe the button's purpose.

### Steps to reproduce

1. Navigate to 'https://app.opensauced.pizza/user/notifications?filter=all'.

### Browsers

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

**Comment by nickytonline** [comment]
Looks good to work on @RitaDee, assigning it to you.
